### ISSUE TITLE
Missing default: users_default_shell

### ISSUE THREAD

**Issue by rikwillems** [debops/ansible-users#38]
The docs mention a default 'users_default_shell' option. This default isn't in the actual package and isn't used in the task:

```
- name: Manage user accounts
  user:
    shell: '{{ item.shell | default(omit) }}'
```

**Comment by drybjed** [comment]
The documentation of `debops.users` role hasn't been converted yet to the new format and is outdated.

I removed the `users_default_shell` variable to not cause unintentional shell changes on accounts. If shell is not specified, the default from `useradd` command will be used.

**Comment by drybjed** [comment]
This issue has been resolved in the `v0.2.0` version.
